src/category_theory/bicategory/functor.lean

```diff
@@ -77,14 +87,15 @@ variables {B}
 /-- Composition of prelax functors. -/
 @[simps]
 def comp : prelax_functor B D :=
-{ map₂ := λ a b f g η, G.map₂ (F.map₂ η), .. F.to_prefunctor.comp G.to_prefunctor }
+{ map₂ := λ a b f g η, G.map₂ (F.map₂ η), .. (F : prefunctor B C).comp ↑G }
```

Comment:
Is the `↑` needed here for this to check? If not, I'd prefer to remove.

Reply:
No, all coercions are used just for readability. All of them can be removed.

Reply:
I may have misunderstood your comment. What I meant in the last reply is that `F.to_prelax_functor.comp G.to_prelax_functor` works. However, `(F: prelax_functor B C) .comp G` doesn't works, so `↑` is actually needed here.

Reply:
Okay!